packages/store-ui/src/ProductQuantity/NumericStepper.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, Fragment, useCallback } from 'react'
+import { Button, Flex, Input } from 'theme-ui'
+
+export interface NumericStepperProps {
+  variant?: string
+  minValue: number | 1
+  maxValue?: number | undefined
+}
+
+var currentQuantity = 1
+
+export const getCurrentQuantity = () => {
+  return currentQuantity
+}
+
+export const NumericStepper: FC<NumericStepperProps> = ({
+  variant = 'productQuantity',
+  minValue,
+  maxValue,
+  children,
+}) => {
+  var [quantity, setQuantity] = React.useState<number>(1)
+
+  const updateQuantity = (value: number) => {
+    currentQuantity = value
+    setQuantity(value)
+  }
+
+  const increaseNumber = useCallback(() => {
+    quantity = quantity + 1
+    if (maxValue) {
+      if (quantity > maxValue) quantity = maxValue
+    }
+    updateQuantity(quantity)
+  }, [quantity, maxValue])
+
+  const decreaseNumber = useCallback(() => {
+    quantity = quantity - 1
+    if (quantity < minValue) quantity = minValue
+    updateQuantity(quantity)
+  }, [quantity, minValue])
+
+  const isMax = maxValue ? quantity >= maxValue : false
+  const isMin = quantity <= minValue
+
+  return (
+    <Fragment>
+      <Flex variant={`${variant}.numericStepper`}>
+        <Button
+          variant={`${variant}.numericStepper.button.minus`}
+          onClick={() => decreaseNumber()}
+          disabled={isMin}
+        >
+          -
+        </Button>
+        <Input
+          value={quantity}
+          onChange={(e) => updateQuantity(Number(e.target.value))}
```

Comment:
I'm not sure if it's a good practice to allow the user to deliberately change the quantity here... It could, for example, set a value greater than max

Reply:
@CarlaCosenza I'd delete this `onChange` here to disable the input. What do you think?

Reply:
You mean not be able to edit the label? I think it would be okay, but the current implementation does it like has this behavior, if I am not mistaken.